src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  width: calc(7 * ($day-square));
+  display: flex;
+  flex-wrap: wrap;
+  counter-reset: day;
+
+  &__day {
+    width: $day-square;
+    height: $day-square;
+    border: 1px solid black;
+    box-sizing: border-box;
+    background-color: #eee;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+    counter-increment: day;
+    padding: 0 $day-padding;
+    transform: translateY(0);
+    transition: transform 0.5s;
+    text-align: center;
+    justify-content: center;
+
+    &::before {
+      content: counter(day);
```

Comment:
Content of the before shouldn't come from counter. In fact you don't need counter at all in this task. This should be done with the [for loop](https://sass-lang.com/documentation/at-rules/control/for/) (see the readme)